src/lib/Libwin/accesinfo.c

```diff
@@ -269,8 +281,7 @@ secure_file(char *path, char *user, ACCESS_MASK mask)
 	si |= DACL_SECURITY_INFORMATION;
 
 	if (SetFileSecurity(path, si, &sd) == 0) {
-		sprintf(logb, "error setting security for file %s", path);
-		log_err(-1, "secure_file", logb);
+		log_err(-1, __func__, "failed in SetFileSecurity");
 		if (usid) LocalFree(usid);
 		if (gsid) LocalFree(gsid);
 		(void)free(pdacl);
```

Comment:
We should add log at https://github.com/PBSPro/pbspro/blob/6514592d1bca52adb5cd2bdb3233be1f9838a25a/src/lib/Libwin/accesinfo.c#L303